llvm/lib/CodeGen/AsmPrinter/AsmPrinter.cpp

```diff
@@ -1348,6 +1348,23 @@ void emitYkBBAddrMapSymbol(MCContext &MCtxt, MCStreamer &OutStreamer,
   OutStreamer.emitLabel(Sym);
 }
 
+/// Returns the fallthrough block of `MBB`, or nullptr if there isn't one.
+MachineBasicBlock *findFallthruBlock(const MachineBasicBlock *MBB) {
+  MachineBasicBlock *FallThru = nullptr;
+  for (MachineBasicBlock *SBB : MBB->successors()) {
```

Comment:
Isn't this clearer as:

```c++
for (...) {
  if (MBB->isLayoutSuccessor(...))
    return SBB;
}
return nullptr;
```

?

Reply:
Hah, yeah. Dunno what i was thinking.

Reply:
Fixed in [8863550](https://github.com/ykjit/ykllvm/pull/50/commits/886355089fe100853a8495aa5f654c68ce812225)